protected/humhub/modules/content/widgets/ScheduleLink.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\content\widgets;
+
+use humhub\libs\Html;
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\content\components\ContentContainerActiveRecord;
+use humhub\modules\content\models\Content;
+use humhub\widgets\Link;
+use Yii;
+use yii\base\Widget;
+
+/**
+ * Schedule link for updating the schedule options of Wall Entries.
+ *
+ * @package humhub.modules_core.wall.widgets
+ * @since 1.14
+ */
+class ScheduleLink extends Widget
+{
+    public ContentActiveRecord $contentRecord;
+
+    /**
+     * @inheritdoc
+     */
+    public function run()
+    {
+        $content = $this->contentRecord->content;
+
+        if ((int) $content->state !== Content::STATE_SCHEDULED) {
```

Comment:
Allow option also for drafts

Reply:
@luke- Done here https://github.com/humhub/humhub/pull/6116/commits/abaa10a0d6c14a094a5f515bc7fb3d02751f5a65#diff-5180ebc7566a79028c0c165f7079fa892457d06405c0ec8a3067fe7f1453f504L35.

But please note currently it is possible only to change state from `draft` to `scheduled`, but cannot revert this back, because if uncheck the option "Enable schedule date time" then content state will be updated to `published`.
Is it ok, if we should implement a selector for a state when the checkbox is unchecked?

Reply:
Hmm, what do you think about: If the user is uncheck the scheduling checkbox on a existing scheduled content, the content will be marked as draft?

Reply:
@luke- Yes, it seems it is more sense to set state to `draft`, ok I will do this.